canvas_sdk/v1/data/condition.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+
+from canvas_sdk.v1.data.base import CommittableModelManager
+from canvas_sdk.v1.data.patient import Patient
+
+
+class Condition(models.Model):
+    class Meta:
+        managed = False
+        app_label = "canvas_sdk"
+        db_table = "canvas_sdk_data_api_condition_001"
+
+    objects = CommittableModelManager()
+
+    id = models.UUIDField()
+    dbid = models.BigIntegerField(primary_key=True)
```

Comment:
Is there a reason for `BigIntegerField` instead of a plain `IntegerField`?

This is very very terribly pedantic of me, but if we're staying in the int range (2147483647)there's no need for the extra 4 bytes, maybe?

Reply:
I believe that's what the source model uses. so just mirroring